db/migrate/20150129082106_add_from_quote_to_topic_link.rb

```diff
@@ -0,0 +1,5 @@
+class AddFromQuoteToTopicLink < ActiveRecord::Migration
+  def change
+    add_column :topic_links, :from_quote, :boolean, default: false, null: false
```

Comment:
There's another problem hiding here - yt-light creates links in javascript that don't show up in the cooked, so they also give the "link not found" white screen.

I now agree, this is the wrong way to fix it.